modules/nw-multus-delete-network.adoc

```diff
@@ -8,6 +8,13 @@ As a cluster administrator, you can remove an additional network from your
 {product-title} cluster. The additional network is not removed from any Pods it
 is attached to.
 
+[IMPORTANT]
+====
+In {product-title} 4.2.0 you must manually delete the additional network CR after removing it from the Cluster Network Operator configuration.
```

Comment:
Are we certain this will apply only to this z-stream release (`4.2.0`), rather than just `4.2`?

Reply:
@bobfuru, it's targeted to be fixed in 4.3.0 and back ported to 4.2.1.